features/aanschrijfwijze-persoon-zonder-partner-met-adellijke-titel-predicaat.feature

```diff
@@ -0,0 +1,356 @@
+        - de adellijke titel en het predicaat wordt opgenomen in de vorm die hoort bij het geslacht van de persoon:
+          | adellijke titel/predicaat | vrouw     | man      | onbekend |
+          | JH                        | jonkvrouw | jonkheer |          |
+          | JV                        | jonkvrouw | jonkheer |          |
+          | R                         |           | ridder   |          |
+          | B                         | barones   | baron    |          |
+          | BS                        | barones   | baron    |          |
+          | H                         | hertogin  | hertog   |          |
+          | HI                        | hertogin  | hertog   |          |
+          | G                         | gravin    | graaf    |          |
+          | GI                        | gravin    | graaf    |          |
+          | M                         | markiezin | markies  |          |
+          | MI                        | markiezin | markies  |          |
+          | P                         | prinses   | prins    |          |
+          | PS                        | prinses   | prins    |          |
+        - De aanspreekvorm wordt op basis van adellijke titel/predicaat en geslacht als volgt samengesteld:
+          | adellijke titel/predicaat | vrouw                    | man                    | onbekend    |
+          | JH                        | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | JV                        | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | R                         | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | B                         | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | BS                        | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | H                         | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | HI                        | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | G                         | De hooggeboren vrouwe    | De hooggeboren heer    |             |
+          | GI                        | De hooggeboren vrouwe    | De hooggeboren heer    |             |
+          | M                         | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | MI                        | De hoogwelgeboren vrouwe | De hoogwelgeboren heer |             |
+          | P                         | De hoogheid              | De hoogheid            | De hoogheid |
+          | PS                        | De hoogheid              | De hoogheid            | De hoogheid |
+
+    Abstract Scenario: persoon zonder voorvoegsel, heeft geslacht "<geslacht>" en een adellijke titel die hoort bij het geslacht
```

Comment:
Ik betwijfel of dit voorkomt, maar kan hier niets over vinden. Zou dit ongewijzigd laten.